pkgs/development/tools/misc/pest-ide-tools/default.nix

```diff
@@ -33,7 +28,7 @@ rustPlatform.buildRustPackage rec {
   meta = with lib; {
     description = "IDE support for Pest, via the LSP";
     homepage = "https://pest.rs";
-    license = with licenses; [ mit asl20 ];
+    license = with licenses; [ asl20 ];
```

Comment:
It's probably not a big deal, but I think I would consider the license change and the dependency change to be two different changes, warranting two different commits.